Added note about Startup Boost blocking auto-open-devtools-for-tabs (#2369)
MicrosoftDocs/edge-developer: microsoft-edge/devtools-guide-chromium/overview.md

   microsoft-edge-dev --auto-open-devtools-for-tabs
   ```
   
> [!IMPORTANT]

> > There must be no running Edge processes when launching via `auto-open-devtools-for-tabs`. You may need to disable Startup Boost in `edge://settings/system`. More information about Startup Boost can be found here: [Get help with startup boose - Microsoft Support](https://support.microsoft.com/topic/get-help-with-startup-boost-ebef73ed-5c72-462f-8726-512782c5e442).

Reviewer: ```suggestion
> There must be no running Edge processes when launching via `auto-open-devtools-for-tabs`. You may need to disable Startup Boost in `edge://settings/system`. For more information, see [Get help with startup boost](https://support.microsoft.com/topic/get-help-with-startup-boost-ebef73ed-5c72-462f-8726-512782c5e442).
```
https://support.microsoft.com/topic/get-help-with-startup-boost-ebef73ed-5c72-462f-8726-512782c5e442

> > [!IMPORTANT]
> > There must be no running Edge processes when launching via `auto-open-devtools-for-tabs`. You may need to disable Startup Boost in `edge://settings/system`. More information about Startup Boost can be found here: [Get help with startup boose - Microsoft Support](https://support.microsoft.com/topic/get-help-with-startup-boost-ebef73ed-5c72-462f-8726-512782c5e442).

Reviewer: ```suggestion

###### Disabling startup boost

> [!IMPORTANT]
> There must be no running Microsoft Edge processes when launching via `auto-open-devtools-for-tabs`. You may need to disable **Startup boost** in `edge://settings/system`. For more information, see [Get help with startup boost](https://support.microsoft.com/topic/get-help-with-startup-boost-ebef73ed-5c72-462f-8726-512782c5e442).
```
Rendered view (GitHub version):
https://github.com/ryanborMSFT/edge-developer/blob/patch-1/microsoft-edge/devtools-guide-chromium/overview.md#have-devtools-automatically-open-when-you-open-a-new-tab

Given the present style sheet, we're moving toward h2/h4/h6.  The containing heading is h3.  h6 should produce a suitable size at LMC (different than GitHub's rendering of heading sizes).

A heading is good to provide a skimmable label for an Alert eg an Important note.

Reviewer: sent chat

> 

Reviewer: Should some of this wording be added from PR main page, for more clarification?

auto-open-devtools-for-tabs requires there to be no running Edge processes. 

Most users have the Startup Boost feature enabled, which keeps a minimal Edge process running in the background. 

This interferes with the auto-open-devtools-for-tabs feature and would need to be disabled to use it.

Or users would have to task-kill the Startup Boost Edge process before running the auto-open-devtools-for-tabs command.
